google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionClient.java

```diff
@@ -223,7 +223,7 @@ DatabaseId getDatabaseId() {
   @Override
   public XGoogSpannerRequestId nextRequestId(long channelId, int attempt) {
     return XGoogSpannerRequestId.of(
-        this.nthId, this.nthRequest.incrementAndGet(), channelId, attempt);
+        this.nthId, channelId, this.nthRequest.incrementAndGet(), attempt);
```

Comment:
The fact that we can make this change without any tests failing (or needing to be changed) seems to indicate that we have a test gap. Will that be covered by the tests that are being added in this PR, but that are currently not verifying anything, as all verifications are in `if (false)` blocks? If not, can we add tests that cover this in a follow-up PR?